packages/lib-classifier/src/plugins/drawingTools/experimental/models/marks/TranscriptionLine/TranscriptionLine.js

```diff
@@ -0,0 +1,86 @@
+import { getParentOfType, types } from 'mobx-state-tree'
+import { TranscriptionLine as TranscriptionLineComponent } from '../../../components/'
+import { TranscriptionLineTool } from '@plugins/drawingTools/models/tools'
+
+import { Mark } from '@plugins/drawingTools/models/marks'
+
+const TranscriptionLineModel = types
+  .model('TranscriptionLineModel', {
+    x1: types.maybe(types.number),
+    y1: types.maybe(types.number),
+    x2: types.maybe(types.number),
+    y2: types.maybe(types.number)
+  })
+  .volatile(self => ({
+    finished: false
+  }))
+  .views(self => ({
+    get coords () {
+      return {
+        x: self.x1,
+        y: self.y1
+      }
+    },
+
+    deleteButtonPosition (scale) {
+      const BUFFER = 16
+      const x = self.x1 + (BUFFER / scale)
+      const y = self.y1
+      return { x, y }
+    },
+
+    get isValid () {
+      return true
+    },
+
+    get tool () {
+      return getParentOfType(self, TranscriptionLineTool)
```

Comment:
Should this change to `getParent` too?

Reply:
Depends whether you want strict typing for the parent tool. The other mark models use `getParentofType` to return a specific type of tool but the generic Mark uses `getParent`.

Reply:
Hm, it seems odd to have the base model potentially have a different function that the child ones, but I don't have a strong opinion on it.